src/tests/services/refreshToken.test.js

```diff
@@ -0,0 +1,43 @@
+import mockAxios from 'axios';
+
+import RefreshToken from 'services/refreshToken';
+import RefreshTokenResponse from 'tests/fixtures/refreshTokenResponse.json';
+
+describe('when refresh token is performed', () => {
+  beforeEach(() => {
+    localStorage.clear();
+  });
+
+  delete window.location;
+  window.location = { reload: jest.fn() };
+
+  it('renders the login page if refresh token is not successful', async () => {
```

Comment:
@rafayet-monon How about
```suggestion
  it('renders the login page if refresh token is missing', async () => {
```